### PR TITLE
Add nowork trigger

### PR DESCRIPTION
**Pull request by JosephSilber**
What do you mean it doesn't work? Is it unemployed?

What happens when you try to run it? What's the output? What's the error message? What did you want or expect to happen?

### CODE REVIEW

**Comment by machuga** [comment]
Can you rephrase it to be a bit more inviting?  We've been trying to have Liona's messages be on the friendlier side lately.

**Comment by JosephSilber** [comment]
Does this sound better?

> What do you mean it doesn't work? What happens when you try to run it? What's the output? What's the error message? What did you want or expect to happen? Please provide more details so that we can properly help you.

A little too soft-gloved for my tastes. But if that's what we're aiming for, so be it.

**Comment by machuga** [comment]
Something more concise maybe like "Saying something doesn't work or is broken is unfortunately unhelpful to those who want to help.  Please provide full error messages, ..." - much of the info in !help can be reused.

**Comment by JosephSilber** [comment]
Whatever. Just close this and do it however you like.

Just pointing out that we should have a standard response to "it doesn't work".

**Comment by machuga** [comment]
Sure.  We typically just use !help for it but I'm all for having a specialized one.  I just want to keep it respectful since we really don't want to turn people away.
